libexec/cli/instance.start.exec

```diff
@@ -181,10 +181,12 @@ if [ -x "$SINGULARITY_libexecdir/singularity/image-handler.sh" ]; then
     . "$SINGULARITY_libexecdir/singularity/image-handler.sh"
 fi
 
+INSTANCE_PROC_NAME="singularity-daemon: $USER [$SINGULARITY_DAEMON_NAME]"
```

Comment:
As we discussed should be `singularity-instance: $USER [$SINGULARITY_DAEMON_NAME]`